modules/apps/object/object-dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/js/MultiSelectPicklist/MultiSelectPicklist.tsx

```diff
@@ -67,6 +67,18 @@ export default function MultiSelectPicklist({
 	value,
 	...otherProps
 }: MultiSelectPicklistProps) {
+	const normalizedValue = normalizeValues(value);
+
+	const onChangeRef = useRef(onChange);
+
+	useEffect(() => {
+		onChangeRef.current = onChange;
+	}, [onChange]);
+
+	useEffect(() => {
+		onChangeRef.current({target: {value: normalizedValue}});
+	}, [normalizedValue]);
+
```

Comment:
not related to this line but about the commit message, you repeated 'to avoid' two times in a row :two:

Reply:
Hi, @rebsilva!, thanks for spotting these issues! Considering that: this is a story PR, CI failures are unrelated and the requested changes (removing a `// @ts-ignore` comment and fixing a commit message) do **not** affect the application, I will forward this to Brian. I can send a follow-up PR removing the `// @ts-ignore` comment and leave the commit message as is, as it is intelligible.

Reply:
okay @cgmonte! no problems :cherry_blossom: